static/scripts/topicEdit.js

```diff
@@ -600,7 +600,9 @@ class TopicResources extends TopicBlock {
         if(!resource) {
             let isCourseGroupTopic = $contentBlocksContainer.data('iscoursegroup') !== undefined;
             // open content search popup
-            const resourcePopup = window.open('/content/?inline=1&isCourseGroupTopic=' + isCourseGroupTopic, "content-search", "toolbar=no, location=no, directories=no, width=800,height=600,status=no,scrollbars=yes,resizable=yes");
+            const resourcePopup = window.open('/content/?inline=1&isCourseGroupTopic=' + isCourseGroupTopic, "content-search",
+				`width=1920, height=1080, fullscreen=yes, toolbar=no, location=no, directories=no, status=no, scrollbars=yes, resizable=yes`);
```

Comment:
Does this also work with other screen resolutions, e.g. Smartphones?

Reply:
Mona told me that when it was previously 800x600 on mobile it was just opening a fullscreen window
so I assume this is the same behaviour

Reply:
If it is bigger than screen it opens in full screen